locales/en.json

```diff
@@ -22,7 +22,7 @@
     "app_location_already_used": "The app '{app}' is already installed on that location ({path})",
     "app_make_default_location_already_used": "Can't make the app '{app}' the default on the domain {domain} is already used by the other app '{other_app}'",
     "app_location_install_failed": "Unable to install the app in this location because it conflit with the app '{other_app}' already installed on '{other_path}'",
-    "app_location_unavailable": "This url is not available or conflicts with an already installed app",
+    "app_location_unavailable": "This url is not available or conflicts with the already installed app(s):\n{apps:s}",
```

Comment:
Hmmm, I never know what's the case where it creates an issue. Isn't this gonna raise a "not all arguments converted during string formatting" error for other translated strings which don't have `{apps}` in them ?

Reply:
Meh, I never remember that neither :/

I'm going to look at the moulinette code, we should fix that there.

Reply:
Yeah, it's gonna failed... https://github.com/YunoHost/moulinette/blob/stretch-unstable/moulinette/core.py#L92